code/modules/judaism/sacrifice.dm

```diff
@@ -0,0 +1,55 @@
+
+/*
+PP Station 13 Foreskin Magic
+Made with love by woross
+*/
+
+
+/obj/structure/jewaltar
+	name = "altar"
+	desc = "An altar, presumably to the Moloch."
+	icon = 'ppstation/icons/jew.dmi'
+	icon_state = "jewaltar"
+	density = TRUE
+	anchored = TRUE
+	layer = TABLE_LAYER
+	climbable = FALSE
+	pass_flags = LETPASSTHROW //You can throw objects over this, despite it's density.")
+	max_integrity = 10000
+	integrity_failure = 30
+	var/foreskincount = 0
```

Comment:
This variable likely doesn't need to exist.
If it's just for an error message, you can create a check on if any foreskins are found on the altar.